images/kube-fluentd-operator/tests/main.tf

```diff
@@ -0,0 +1,48 @@
+terraform {
+  required_providers {
+    oci    = { source = "chainguard-dev/oci" }
+    helm   = { source = "hashicorp/helm" }
+    random = { source = "hashicorp/random" }
+  }
+}
+
+variable "skip_crds" {
+  description = "Used to deconflict between multiple installations within the same cluster."
+  default     = false
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "kfo" {
+  name             = "kfo-${random_pet.suffix.id}"
+  namespace        = "kfo"
+  chart            = "https://github.com/vmware/kube-fluentd-operator/releases/download/v1.17.4/log-router-0.4.0.tgz" # not sure how we can get the correct version of the chart rather than hardcoding a version below
```

Comment:
this is the one thing I'm not sure about, it works for now but maintaining a version here isn't going to work really.  I couldn't figure out a combination of github `/latest` or `/release` that would work.

Reply:
Let's get this in and figure something out later when this breaks us.